src/org/traccar/model/Position.java

```diff
@@ -39,6 +39,8 @@
     public static final String KEY_LAC = "lac";
     public static final String KEY_CID = "cid";
     public static final String KEY_FUEL = "fuel";
+    public static final String KEY_FUEL_CONSUMPTION = "fuelConsumption";
+    public static final String KEY_FUEL_INSTANT_CONSUMPTION = "fuelInstantConsumption";
```

Comment:
Most stored attributes are there because someone asked to add it. I think we can remove this one because the one protocol that provides it, also provides regular consumption as well.